include/util/edit.h

```diff
@@ -52,12 +56,27 @@ class EditEngine {
   data::BinData originalBytesValues(size_t offset, size_t size) const;
 
  private:
+  struct EditNode {
+    std::shared_ptr<data::BinData> fragment_;
```

Comment:
Why shared_ptr? It doesn't seem to be shared with anything outside of this object.